src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -559,6 +559,33 @@ internal static void AsAnyCleanupNative(IntPtr address, object o)
             Marshal.DestroyStructure(address, o.GetType());
         }
 
+        internal static unsafe object? OleVariantToObject(IntPtr pSrcNativeVariant)
```

Comment:
Nit: I would call this `Variant` like what it is called in most other places in .NET.